src/items/items.component.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Box, Button } from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
+import ItemDialog from './itemDialog.component';
+import { useCatalogueItem } from '../api/catalogueItem';
+import { Link, useLocation } from 'react-router-dom';
+import { useCatalogueCategory } from '../api/catalogueCategory';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+
+export function Items() {
+  const [addItemDialogOpen, setAddItemDialogOpen] =
+    React.useState<boolean>(false);
+  const location = useLocation();
+  const catalogueItemId = location.pathname.split('/')[3];
+  const { data: catalogueItem } = useCatalogueItem(catalogueItemId);
+  const { data: catalogueCategory } = useCatalogueCategory(
+    catalogueItem?.catalogue_category_id
+  );
+  return (
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'left',
+        padding: 4,
+        position: 'sticky',
+        top: 0,
+        backgroundColor: 'background.default',
+        zIndex: 1000,
+        width: '100%',
+      }}
+    >
+      <Button
+        sx={{ mx: 0.5 }}
+        component={Link}
+        startIcon={
+          <Box sx={{ alignItems: 'center', display: 'flex' }}>
+            <ArrowBackIcon fontSize="small" />{' '}
+            <ArrowBackIcon fontSize="small" />
```

Comment:
![image](https://github.com/ral-facilities/inventory-management-system/assets/144466888/371b054b-46a2-434c-999e-afe1f8c8786c)
Double arrow makes it look like this. Shouldn't it be a single arrow?

Reply:
It meant to be a double as you are going back up two levels